Src/Web/Web.Shared.Net/RequestTrackingTelemetryModule.cs

```diff
@@ -19,12 +20,16 @@
     /// </summary>
     public class RequestTrackingTelemetryModule : ITelemetryModule
     {
-        private readonly IList<string> handlersToFilter = new List<string>();
+        private const string CorrelationStatusHeartbeatProperty = "CorrelationHealthStatus_RequestWithContent";
```

Comment:
Lets use the minimum required characters. "CorHS_ReqWCont" 
"NA" instead of notsupported :) to minimize payload size